packages/bridge-ui-v2/src/styles/base.css

```diff
@@ -17,9 +17,24 @@
   html,
   body {
     height: 100%;
-    background-image: url(/bg/grid.svg), url(/bg/spotlights.svg), linear-gradient(270deg, #1f1f1f 0%, #000000 100%);
+  }
+
+  body {
+    /* No grid background by default */
+    background-image: url(/bg/spotlights.svg), linear-gradient(180deg, rgb(11 16 27 / 70%) 0%, rgb(11 16 27 / 0%) 100%);
     background-size: 840px, cover, cover;
     background-blend-mode: color-dodge, lighten, multiply;
     background-attachment: fixed;
   }
+
+  @media (min-width: 768px) {
+    body {
+      /* Desktop of larger screens (md) have grid */
+      /* TODO: Ask Jane if we have the right svgs here */
+      background-image:
+        url(/bg/grid.svg),
+        url(/bg/spotlights.svg),
+        /* grey-900/70 => grey-900/0 */ linear-gradient(180deg, rgb(11 16 27 / 70%) 0%, rgb(11 16 27 / 0%) 100%);
+    }
+  }
```

Comment:
```suggestion
  body {
    /* No grid background by default */
    height: 100%;
    background-image: url(/bg/grid.svg), url(/bg/spotlights.svg), linear-gradient(270deg, #1f1f1f 0%, #000000 100%);
    background-size: 0, cover, cover; /* by setting the first image size to 0 for mobile, we hide the grid but keep the blend effect */
    background-blend-mode: color-dodge, lighten, multiply;
    background-attachment: fixed;
  }

  @media (min-width: 768px) {
    body{
      background-size: 840px, cover, cover;
    }
  }
```

Reply:
We still have the problem of hiding the grid for mobile. We need to backgrounds here then